sdk/dependencymap/Azure.ResourceManager.DependencyMap/api/Azure.ResourceManager.DependencyMap.net8.0.cs

```diff
@@ -0,0 +1,343 @@
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.DependencyMap.Models.DependencyMapDateTimeFilter>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    public partial class DependencyMapDiscoverySourcePatch : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.DependencyMap.Models.DependencyMapDiscoverySourcePatch>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.DependencyMap.Models.DependencyMapDiscoverySourcePatch>
+    {
+        public DependencyMapDiscoverySourcePatch() { }
+        public System.Collections.Generic.IDictionary<string, string> Tags { get { throw null; } }
+        protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.DependencyMap.Models.DependencyMapDiscoverySourcePatch System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.DependencyMap.Models.DependencyMapDiscoverySourcePatch>.Create(ref System.Text.Json.Utf8JsonReader reader, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        void System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.DependencyMap.Models.DependencyMapDiscoverySourcePatch>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.DependencyMap.Models.DependencyMapDiscoverySourcePatch System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.DependencyMap.Models.DependencyMapDiscoverySourcePatch>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.DependencyMap.Models.DependencyMapDiscoverySourcePatch>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.DependencyMap.Models.DependencyMapDiscoverySourcePatch>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    public abstract partial class DependencyMapDiscoverySourceProperties : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.DependencyMap.Models.DependencyMapDiscoverySourceProperties>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.DependencyMap.Models.DependencyMapDiscoverySourceProperties>
+    {
+        protected DependencyMapDiscoverySourceProperties(string sourceId) { }
+        public Azure.ResourceManager.DependencyMap.Models.DependencyMapProvisioningState? ProvisioningState { get { throw null; } }
+        public string SourceId { get { throw null; } set { } }
```

Comment:
can this type be more specific?

Reply:
saw there is a type alternateen in spec PR, not working here?

Reply:
fixed